modules/virt-migrating-storage-class-ui.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * virt/managing_vms/virtual_disks/virt-migrating-storage-class.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="virt-migrating-storage-class-ui_{context}"]
+= Migrating VM disks to a different storage class by using the web console
+
+You can migrate one or more disks attached to a virtual machine (VM) to a different storage class by using the {product-title} web console. When performing this action on a running VM, the operation of the VM is not interrupted and the data on the migrated disks remains accessible.
+
+[NOTE]
+====
+With the {VirtProductName} Operator, you can only start storage class migration for one VM at the time and the VM must be running. If you need to migrate more VMs at once or migrate a mix of running and stopped VMs, consider using the link:https://docs.redhat.com/en/documentation/migration_toolkit_for_containers/{mtc-version}/html/migration_toolkit_for_containers/index[{mtc-first}].
+
+{mtc-full} is not part of {VirtProductName} and requires separate installation.
+====
+
+.Prerequisites
+
+* You must have a data volume or a persistent volume claim (PVC) available for storage class migration.
+* The cluster must have a node available for live migration. As part of the storage class migration, the VM is live migrated to a different node.
+* The VM must be running.
+
+.Procedure
+
+. Navigate to *Virtualization* -> *VirtualMachines* in the web console.
+. Click the Options menu {kebab} beside the virtual machine and select *Migration* -> *Storage*. You can also access this option from the *VirtualMachine details* page by selecting *Actions* -> *Migration* -> *Storage*.
+. On the *Migration details* page, choose whether to migrate the entire VM storage or selected volumes only. If you click *Selected volumes*, select all disks you intend to migrate. Click *Next* to proceed.
+. On the *Destination StorageClass* page, select the storage class to migrate to from the list of available options. Click *Next* to proceed.
```

Comment:
I can follow this, but "to migrate to from" takes a bit of work to parse. Would this work?
```suggestion
. From the list of available options on the *Destination StorageClass* page, select the storage class to migrate to. Click *Next* to proceed.
```